core/src/main/java/com/netflix/conductor/validations/WorkflowTaskTypeConstraint.java

```diff
@@ -271,19 +270,6 @@ private boolean isWaitTaskValid(
                 context.buildConstraintViolationWithTemplate(message).addConstraintViolation();
                 valid = false;
             }
-
-            try {
-                if (StringUtils.isNotBlank(duration)) {
-                    DateTimeUtils.parseDuration(duration);
-                } else if (StringUtils.isNotBlank(until)) {
-                    DateTimeUtils.parseDate(until);
-                }
-            } catch (Exception e) {
```

Comment:
Please update this to catch the correct exceptions from the methods, `NumberFormatException`, `IllegalArgumentException` and `ParseException`.